README.rst

```diff
@@ -17,15 +18,14 @@ Instalar as seguintes dependências do sistema::
 
 * ::
 
-    sudo apt-get install git nginx python3-dev libpq-dev graphviz-dev graphviz \
-    pkg-config postgresql postgresql-contrib pgadmin3 python-psycopg2 \
-    software-properties-common build-essential libxml2-dev libjpeg-dev \
-    libssl-dev libffi-dev libxslt1-dev python3-setuptools curl
+    sudo apt-get install git python3-dev libpq-dev graphviz-dev graphviz \
+    pkg-config software-properties-common build-essential libxml2-dev \
+    libjpeg-dev libssl-dev libffi-dev libxslt1-dev python3-setuptools curl
 
     sudo easy_install3 pip lxml
 
     sudo -i
-    curl -sL https://deb.nodesource.com/setup_5.x | bash -
+    curl -sL https://deb.nodesource.com/setup_666666bash -
```

Comment:
Sim, devia ser 6.x | bash -

On Thu, Sep 8, 2016 at 9:28 AM, Leandro Roberto da Silva <
notifications@github.com> wrote:

> In README.rst
> <https://github.com/interlegis/saap/pull/9#discussion_r77995650>:
>
> >
> >      sudo easy_install3 pip lxml
> >
> >      sudo -i
> > -    curl -sL https://deb.nodesource.com/setup_5.x | bash -
> > +    curl -sL https://deb.nodesource.com/setup_666666bash -
>
> isso é um erro, certo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/interlegis/saap/pull/9/files/31b1a6d413b5009fe49b7f94b27f79759df53b31#r77995650>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAR5NiANngt26ymKH6kzPsVQ2fb-mveLks5qn_9vgaJpZM4J3hUG>
> .
>



-- 
--
-- Ramiro Batista da Luz
-- (41) 9173-2231
-- http://www.ramiroluz.eti.br